src/components/App.tsx

```diff
@@ -17,12 +17,13 @@ const App = () => (
     <Welcome />
     <Description />
     <StickyOverlappingSections />
-    <JoinCommunity />
+    <JoinNetwork />
     <SignUpBelow />
     <Slider />
     <KeepingSecure />
     <SafetyInfo />
     <SecurityProtocol />
+    {/* <Partners /> */}
```

Comment:
Avoid comments.... we can also return it later on; You'll notice the import isn't there on top either; so, we end up in a sub-optimal state. 

You will notice you also updated the Partners.tsx 

Instead of a comment, in this particular case can you add a feature flag? (check for  a `partners` query parameter in the url, and show the section if it is there)

Reply:
It is commented since it should be back on shortly. It also avoids trying to do the same components by another programist if i wont be when it is back on.
Could you tell me more about this flag? It sounds like something you would not like here since either Partners should be always on or always off, not conditional.

Reply:
Yes, but feature flags are a way to enable something conditionally until you make them always on

https://launchdarkly.com/blog/what-are-feature-flags/

Reply:
I understand what conditional rendering is, my question was about necessity of such rendering :D
But i already did what u asked for, after adding "?partners" to url the component will render.